flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/casting/CastRulesTest.java

```diff
@@ -76,7 +155,77 @@
                                         Thread.currentThread().getContextClassLoader()),
                                 TimestampData.fromLocalDateTime(
                                         LocalDateTime.parse("2021-09-24T12:34:56.123456")),
-                                StringData.fromString("2021-09-24 14:34:56.123456")),
+                                StringData.fromString("2021-09-24 14:34:56.123456"))
```

Comment:
Maybe you could extract those dates/time/timestamps to static vars, similar to `CastFunctionITCase` so we can always have those default tests, and then use some other date/time values if needed to test some other cases, or edge cases.

Reply:
Check now

Reply:
thank you!